en/telemetry/esp8266_wifi_module.md

```diff
@@ -92,30 +93,34 @@ If you have any problem connecting, see [QGC Installation/Configuration Problems
 :::
 
 
-## Module Flashing/Setup (Advanced)
-
-The module should come preconfigured with appropriate firmware.
-If not, you can use the instructions below to update it with the latest version.
-
-### Build From Sources
-
-The [firmware repository](https://github.com/dogmaphobic/mavesp8266) contains instructions and all the tools needed for building and flashing the firmware.
+## ESP8266 Flashing/Firmware (Advanced)
 
+ESP8266 modules from different manufacturers may not have appropriate firmware pre-installed.
+The instructions below explain how to update it with the latest version.
 
 ### Pre Built Binaries
 
 [MavLink ESP8266 Firmware V 1.2.2](http://www.grubba.com/mavesp8266/firmware-1.2.2.bin)
 
-### Updating the Firmware
+### Build From Sources
+
+The [firmware repository](https://github.com/dogmaphobic/mavesp8266) contains instructions and all the tools needed for building and flashing the firmware.
+
+### Updating the Firmware OTA
```

Comment:
@bkueng @teyrana my understanding is that this OTA both uploads and installs the new firmware - is that correct (i.e. if you have a recent firmware, you would always pick this option for an update). If so, then might reorg a little to make this more clear.

Reply:
(note, I am merging this anyway, so this would be a post process)

Reply:
I never tried that as I wasn't aware of it. Would make things easier though.

Reply:
I never flashed my radios as I didn't need to.  Both worked with a little configuration.

Reply:
Also, I think this phrasing is a bit ambiguous... this page is discussing how to flash firmware _for the telemetry_ radio, and in the middle of that, you discuss flashing firmware _for the FMU_.  

At very least, each mention of firmware explicitly mentioned which one you're referring to.

Reply:
@teyrana Thanks, good point, agreed. Done in https://github.com/PX4/PX4-user_guide/pull/1203

I also added the images as part of the note explaining that details may be written on the board.